client/task-list/placeholder.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+const TaskListPlaceholder = ( props ) => {
+	const {
+		numTasks = 5,
+	} = props;
+	return (
+		<div className="woocommerce-task-dashboard__container">
```

Comment:
How about this class name?

```suggestion
		<div className="woocommerce-task-list__container">
```

Reply:
That's the existing class name here: https://github.com/woocommerce/woocommerce-admin/blob/master/client/dashboard/task-list/index.js#L373

Reply:
That's the existing class name here: https://github.com/woocommerce/woocommerce-admin/blob/master/client/dashboard/task-list/index.js#L373